api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -31,27 +35,29 @@
 import org.pmiops.workbench.db.model.DbWorkspaceFreeTierUsage;
 import org.pmiops.workbench.mail.MailService;
 import org.pmiops.workbench.model.BillingStatus;
+import org.pmiops.workbench.utils.CostComparisonUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
 
 /** Methods relating to Free Tier credit usage and limits */
 @Service
 public class FreeTierBillingService {
 
+  public static final Long TWO_DAYS = 172_800_000l;
```

Comment:
Yes I agree, since it's my first change to this codebase, I didn't know what to expect so I chose to be extra cautious. I'll drop this TWO_DAYS for now, and just adjust the config in test env